content/en/api/embeds/embeds_get.md

```diff
@@ -21,13 +21,5 @@ Returns: A JSON object with 8 elements:
 On failure, the return value is a JSON containing an error message {errors: [messages]}.
 
 ##### ARGUMENTS
-* **`size`** [*optional*, *default*=**medium**]:  
-    The size of the graph. Must be one:
-    * **small**, 
-    * **medium**, 
-    * **large**, 
-    * **xlarge**.
-* **`legend`** [*optional*, *default*=**no**]:  
-    The flag determining if the graph includes a legend. Must be one of **yes** or **no**.
-* **`template_variables`** [*optional*, *default*=**None**]:  
-    Replace template variables in queries with form $var. To replace $var with val, use var=val as a parameter for each template variable you wish to replace. If any template variables are missing values in the iframe source url, then (*) is used as the value.
+
+This endpoint takes no JSON arguments.
```

Comment:
How can you get an specific embed without at least specifying the id?

Reply:
The embed ID is in the URL directly: https://docs.datadoghq.com/api/?lang=python#get-specific-embed